bcal.c

```diff
@@ -715,7 +974,7 @@ int main(int argc, char **argv)
 		}
 	}
 
-	if (argc - optind == 1 || (argc == 1 && optind == 1)) {
+	if ((argc - optind == 1 && strlen(argv[1]) < 5) || (argc == 1 && optind == 1)) {
```

Comment:
Why the hardcoded value 5? Can't we do `a + b + c - d` ...? Now that we have a nice parser it shouldn't be a problem, right?